### PR TITLE
Add DCS Service

### PR DESCRIPTION
**Pull request by zhongjun2**

### CODE REVIEW

File: opentelekomcloud/resource_opentelekomcloud_dcs_instance_v1_test.go [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+			},
+		},
+	})
+}
+
+func testAccCheckDcsV1InstanceDestroy(s *terraform.State) error {
+	config := testAccProvider.Meta().(*Config)
+	dcsClient, err := config.dcsV1Client(OS_REGION_NAME)
+	if err != nil {
+		return fmt.Errorf("Error creating Flexibleengine instance client: %s", err)
```
Comment (niuzhenguo) [inline opentelekomcloud/resource_opentelekomcloud_dcs_instance_v1_test.go]: s/FlexibleEngine/OpenTelekomCloud

File: opentelekomcloud/resource_opentelekomcloud_dcs_instance_v1_test.go [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+		}
+
+		if rs.Primary.ID == "" {
+			return fmt.Errorf("No ID is set")
+		}
+
+		config := testAccProvider.Meta().(*Config)
+		dcsClient, err := config.dcsV1Client(OS_REGION_NAME)
+		if err != nil {
+			return fmt.Errorf("Error creating Flexibleengine instance client: %s", err)
```
Comment (niuzhenguo) [inline opentelekomcloud/resource_opentelekomcloud_dcs_instance_v1_test.go]: s/FlexibleEngine/OpenTelekomCloud

File: opentelekomcloud/resource_opentelekomcloud_dcs_instance_v1_test.go [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+		}
+
+		v, err := instances.Get(dcsClient, rs.Primary.ID).Extract()
+		if err != nil {
+			return fmt.Errorf("Error getting Flexibleengine instance: %s, err: %s", rs.Primary.ID, err)
```
Comment (niuzhenguo) [inline opentelekomcloud/resource_opentelekomcloud_dcs_instance_v1_test.go]: ditto.

File: opentelekomcloud/resource_opentelekomcloud_dcs_instance_v1_test.go (hunk 2 above)
Comment (zhongjun2) [reply opentelekomcloud/resource_opentelekomcloud_dcs_instance_v1_test.go]: DONE

File: opentelekomcloud/resource_opentelekomcloud_dcs_instance_v1_test.go (hunk 1 above)
Comment (zhongjun2) [reply opentelekomcloud/resource_opentelekomcloud_dcs_instance_v1_test.go]: DONE

**Pull request merged**